src/MetaModels/MetaModel.php

```diff
@@ -994,11 +995,14 @@ protected function createNewItem($item)
 
     /**
      * {@inheritdoc}
+     *
+     * @SuppressWarnings(PHPMD.Superglobals)
      */
     public function saveItem($objItem)
     {
+        $editInformation = $GLOBALS['container']['dc-general.edit-information'];
```

Comment:
I don't like it at all. We shouldn't add hidden dependencies. The MM or Item interface should provide a way to handle it.

I suggest following. As we are still in alpha, we could break the API without bumping the major version.

```php

public function saveItem($objItem, $setTimestamp = true);

```

Then the DCG data provider could handle it. 

After merging this PR it wouldn't be possible to use MM outside of the DCG context!

I'm also not sure if the edit information class is needed at all. The DCG models provide a way for meta data. But let's discuss it in the DCG related PR.